M/MSTM3/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MSTM3"
+version = v"3.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://www.eng.auburn.edu/~dmckwski/scatcodes/mstm%20v3.0.zip", "ed3b046ef80bc3260849cb7bd2d5629aedf386c0f31bf7a9dc347c8a8e31ee9b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+if [[ "$target" == *-mingw* ]]; then
+    cd ${includedir}
+    cp ${prefix}/src/mpi.f90 .
```

Comment:
Why moving to `${includedir}`?  Can't you do the same from the source directory?  This would save you from having to clean up `${includedir}` at the end.

Reply:
Resolved by adding -I${includedir}